obspy/core/event/catalog.py

```diff
@@ -497,8 +497,8 @@ def write(self, filename, format, **kwargs):
             format_ep = EVENT_ENTRY_POINTS_WRITE[format]
             # search writeFormat method for given entry point
             write_format = buffered_load_entry_point(
-                format_ep.dist.key, 'obspy.plugin.event.%s' % (format_ep.name),
-                'writeFormat')
+                format_ep.dist.name,
+                'obspy.plugin.event.%s' % (format_ep.name), 'writeFormat')
```

Comment:
Are we able to change all of these to f-strings? There are quite a few of these, and they're all a bit different, such as this one having superfluous parentheses.